src/app/dataset-create/dataset-create.component.html

```diff
@@ -30,11 +30,11 @@ <h1 class="title">Create a new dataset</h1>
             </div>
         </div>
         <mat-divider></mat-divider>
-        <div class="my-4">
+        <div class="switch-kind-container my-4">
             <label class="d-flex" for="kind-root">
                 <input name="kind" type="radio" value="root" formControlName="kind" />
```

Comment:
Please add the `id` same as the attribute `for` on the label
```suggestion
                <input name="kind" type="radio" value="root" id="kind-root" formControlName="kind" />
```

Reply:
Added